src/modules/common/renderHtml.js

```diff
@@ -152,7 +152,7 @@ export default (html: string, files: FileCacheStateType, theme: ThemeType) => {
     }
   </style>
 </head>
-<body>
+<body dir="${direction}">
```

Comment:
Is this really required here? What happens to arabic/german text when this is not set?

As far as I understand unicode this should not be needed. Maybe I'm wrong. If not then propably an other fix is needed.

Reply:
This is absolutely necessary here (for example for setting the direction for images, tables, etc.). We do that in the webapp as well.